api/mapping/models.py

```diff
@@ -211,6 +213,31 @@ def __str__(self):
         return str(self.id)
 
 
+class ScanReportConcept(BaseModel):
```

Comment:
The only field "missing" here is a field to record the string we send to NLP:

`processed_string = models.CharField(max_length=64)`

Reply:
I've added it, but I think we might need to come back to it later and discuss a better place to store this information.